src/imv.c

```diff
@@ -760,10 +765,14 @@ static void handle_event(struct imv *imv, SDL_Event *event)
   }
 
   switch(event->type) {
-    case SDL_QUIT:
-      imv_command_exec(imv->commands, "quit", imv);
+    case SDL_QUIT: {
+      /* new scope needed in order to declare variables in a switch statement */
```

Comment:
Another reason to keep `imv_command_exec` around, and have a wrapper for multiple commands :)